pybess/bess.py

```diff
@@ -393,6 +393,19 @@ def destroy_port(self, name):
         request.name = name
         return self._request('DestroyPort', request)
 
+    def set_port_config(self, name, arg):
+        request = bess_msg.SetPortConfRequest()
+        request.name = name
+        request.conf.mac_addr = arg.pop('mac_addr', '00:00:00:00:00:00')
```

Comment:
a single comment line would be nice, something like "no update is done if mac_addr or mtu is not specified".